spyder/plugins/application/container.py

```diff
@@ -296,23 +309,34 @@ def _check_updates_ready(self):
                         "<code>conda update anaconda</code><br>"
                         "<code>conda install spyder={}</code><br><br>"
                     ).format(latest_release)
-                else:
+                elif is_pynsist() or running_in_mac_app():
+                    box.setStandardButtons(QMessageBox.Yes |
+                                           QMessageBox.No)
                     content = _(
-                        "Click <a href=\"{}\">this link</a> to "
-                        "download it.<br><br>"
-                    ).format(url_r)
+                        "Would you like to automatically download and"
+                        " install the latest version of Spyder?<br><br>"
```

Comment:
```suggestion
                        "Would you like to automatically download and "
                        "install it?<br><br>"
```

We already say "Spyder X.Y.Z is available" in the previous sentence, so it sounds strange, confusing and repetitive to say "the latest version of Spyder" here. Also, fix whitespace to be more conventional.